docs/reference/ilm/getting-started-ilm.asciidoc

```diff
@@ -3,77 +3,87 @@
+. <<ilm-gs-bootstrap, Bootstrap the initial time-series index>> by explicitly designating it as the write index.
+. <<ilm-gs-check-progress, Verify that indexes are moving through the lifecycle phases>> as expected with the 
+you use the {ilm-init} explain API. 
 
 [float]
 [[ilm-gs-create-policy]]
-=== Setting up a policy
+=== Create a lifecycle policy
 
-There are many new features introduced by {ilm-init}, but we will only focus on
-a few that are needed for our example. For starters, we will use the
-<<ilm-put-lifecycle,Put Policy>> API to define our first policy. Lifecycle
-policies are defined in JSON and include specific
-<<ilm-policy-definition,phases and actions>>.
+In your lifecycle policy, you specify the phases in your index lifecycle
+and the actions to perform in each phase. A lifecycle can have up to four  phases:
+`hot`, `warm`, `cold`, and `delete`. Policies are defined in JSON 
+and added through the {ilm-init} Put policy API. 
+
+For example, the following request creates a `data_stream` policy with two phases:
+ 
+* The `hot` phase defines a `rollover` action to specify that an index rolls over when it 
+reaches either a `max_size` of 50 gigabytes or a `max_age` of 30 days.
+* The `delete` phase uses `min_age` to move an index into the
+delete phase when it is 90 days old and immediately triggers the  `delete` action to remove the index.
 
 [source,console]
 ------------------------
-PUT _ilm/policy/datastream_policy   <1>
+PUT _ilm/policy/datastream_policy 
 {
-  "policy": {                       <2>
+  "policy": {                       
     "phases": {
-      "hot": {                      <3>
+      "hot": {                      <1>
         "actions": {
-          "rollover": {             <4>
-            "max_size": "50GB",
+          "rollover": {             
+            "max_size": "50GB",     <2>
             "max_age": "30d"
           }
         }
       },
       "delete": {
-        "min_age": "90d",           <5>
+        "min_age": "90d",           <3>
         "actions": {
-          "delete": {}              <6>
+          "delete": {}              <4>
         }
       }
     }
   }
 }
 ------------------------
 
-<1> call to the <<ilm-put-lifecycle,put lifecycle API>> endpoint to create
-    a new policy named "datastream_policy"
-<2> policy definition sub-object
-<3> the hot phase defined in the "phases" section. Optional `min_age` field
-    not defined -- defaults to `0ms`
-<4> rollover action definition
-<5> delete phase begins after 90 days
-<6> delete action definition
 
+<1> The `min_age` field defaults to `0ms` in the hot phase, so new indices immediately enter the hot phase. 
+<2> The `rollover` action is triggered when any of the conditions are met.
+<3> An index is moved into the delete phase after 90 days.
+<4> No additional conditions are specified, so the `delete` action is triggered when the index enters the delete phase. 
```

Comment:
the delete action doesn't support any conditionals/parameters so I think we shouldn't mention conditionals here but rather "The delete action is triggered ..."

Reply:
Fixed.